pkg/kubelet/dockertools/exec.go

```diff
@@ -45,14 +49,21 @@ func (*NsenterExecHandler) ExecInContainer(client DockerInterface, container *do
 		return fmt.Errorf("exec unavailable - unable to locate nsenter")
 	}
 
+	cgexec, err := exec.LookPath("cgexec")
```

Comment:
Let's use the `CgroupManager` interface in `pkg/kubelet/cm` package for this purpose. It might have to be extended.
I do not like adding dependencies.